api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -796,6 +796,189 @@
+	</changeSet>
+	<changeSet author="vshankar" id="201610242135-3">
+        <sql>
+			UPDATE cohort_member
+			INNER JOIN cohort USING (cohort_id)
+			SET cohort_member.start_date = cohort.date_created;
+		</sql>
+	</changeSet>
+	<changeSet author="vshankar" id="201610242135-3.1">
+		<addNotNullConstraint tableName="cohort_member" columnName="start_date" columnDataType="DATETIME"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-4">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="end_date"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="end_date" type="datetime"/>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-5">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="creator"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="creator" type="int"/>
+		</addColumn>
+		<addForeignKeyConstraint baseColumnNames="creator" baseTableName="cohort_member" constraintName="cohort_member_creator" deferrable="false" initiallyDeferred="false" referencedColumnNames="user_id" referencedTableName="users"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-5.1">
+		<sql>
+			UPDATE cohort_member
+			SET cohort_member.creator = 1;
+		</sql>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-5.2">
+		<addNotNullConstraint tableName="cohort_member" columnName="creator" columnDataType="int"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-6">
+		<preConditions>
+			<not>
+				<primaryKeyExists tableName="cohort_member"/>
+			</not>
+		</preConditions>
+		<addPrimaryKey tableName="cohort_member" columnNames="cohort_member_id"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171246-7">
+		<addAutoIncrement tableName="cohort_member" columnName="cohort_member_id" columnDataType="int"/>
+	</changeSet>
+	<changeSet id="201610042145-1" author="vshankar">
+        <preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="date_created"/>
+			</not>
+		</preConditions>
+        <addColumn tableName="cohort_member">
+			<column name="date_created" type="DATETIME"/>
+		</addColumn>
+	</changeSet>
+	<changeSet id="201610042145-1.1" author="vshankar">
+		<sql>
+			UPDATE cohort_member
+			INNER JOIN cohort USING (cohort_id)
+			SET cohort_member.date_created = cohort.date_created;
+		</sql>
+	</changeSet>
+	<changeSet id="201610042145-1.2" author="vshankar">
+		<addNotNullConstraint tableName="cohort_member" columnName="date_created" columnDataType="DATETIME"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201610131530-1">
+		<preConditions>
+			<not>
+				<columnExists tableName="cohort_member" columnName="voided"/>
+			</not>
+		</preConditions>
+        <addColumn tableName="cohort_member">
+			<column name="voided" type="BOOLEAN">
+				<constraints nullable="false"/>
```

Comment:
Alternatively instead of having separate changesets to set new column values, you can set the default value, in this case you set it with the `defaultValueBoolean` attribute